src/main/java/org/edumips64/utils/cli/RunCommand.java

```diff
@@ -0,0 +1,44 @@
+package org.edumips64.utils.cli;
+
+import org.edumips64.core.is.HaltException;
+import org.edumips64.utils.CurrentLocale;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.ParentCommand;
+
+@Command(name = "run", resourceBundle = "CliMessages")
+public class RunCommand implements Runnable {
+
+    @ParentCommand
+    private Cli cli;
+
+    @Override
+    public void run() {
+        if (cli.isCpuRunnable()) {
+            execute();
+        } else {
+            cli.printNotRunnable();
+        }
+    }
+
+    private void execute() {
+        int steps = 0;
+        long startTimeMs = System.currentTimeMillis();
+        try {
+            while(true) {
```

Comment:
Programs can run for thousands of steps or more.

1. can you please add some sort of visual feedback to the user? For example, a dot every 100 steps or so?
2. how can the user stop the execution? Do they use SIGINT or other signals? If so, what happens? Do we want the emulator to retain control of the execution (to allow the user to inspect what happened) or do we want the program to terminate?

I think point 2 can be considered out of scope for this PR, but I'd prefer you to address point 1 (visual feedback) if possible.

Reply:
@lupino3 will resolve that today or tomorrow and push up the change.